src/miner.cpp

```diff
@@ -228,6 +234,13 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript &sc
         LOGA("CreateNewBlock(): total size %llu txs: %llu fees: %lld sigops %u\n", nBlockSize, nBlockTx, nFees,
             nBlockSigOps);
 
+        // sort tx if there are any and the feature is enabled
+        if (enableCanonicalTxOrder.Value() && pblock->vtx.size() > 1)
```

Comment:
I know this is WIP but was wondering why you don't hook up CTOR validation to nov 15 2018 activation code via helpers like `IsNov152018Enabled(Params().GetConsensus(), chainActive.Tip()))`

Reply:
It is hooked up.  The reason for the tweak is to allow manual override as per BUIP098